data/common.yaml

```diff
@@ -57,6 +57,7 @@ mcollective::plugin_classes:
 mcollective::server: true
 mcollective::client: false
 mcollective::purge: true
+mcollective::manage_facts_refresh: true
```

Comment:
This should be removed.

Reply:
@smortex Good catch, this is now removed.

Reply:
@treydock forgot the `-f` while git pushing :sweat_smile:?  The diff still include these bits.

Reply:
@smortex 🤦 Too many remotes, I pushed it to local GHE and not Github.  This should now be fixed.